l1-contracts/scripts/deploy-erc20.ts

```diff
@@ -27,22 +27,27 @@ type TokenDescription = Token & {
   implementation?: string;
 };
 
-async function deployToken(token: TokenDescription, wallet: Wallet): Promise<Token> {
+async function deployToken(token: TokenDescription, wallet: Wallet, nonce?: number): Promise<[Token, number]> {
```

Comment:
please add a comment and mention what this method returns.

Also mention, that it does its internal nonce management, so no other transactions should be sent from this address while this method is running.